### ISSUE TITLE
Unable to upload photos to Google Blogger

### ISSUE THREAD

**Issue by drhamptn** [OpenLiveWriter/OpenLiveWriter#608]
For several months, I've been getting 'The remote server returned an error: (500) Internal Server Error' when uploading posts containing .jpg photos (even one).  I have checked google Photos storage space and permissions (lots of space left), and loading photos through the online Google blog editor (it works).  Posts without photos go up fine, but if i then edit the post in OLW, I get the same error.  I have renewed the blog template, removed and added the account several times, disabled my antivirus and google photos, and re-installed the app after a clean uninstall, without effect.

'any suggestions?

**Comment by ScottIsAFool** [comment]
@drhamptn Can you please follow the instructions here http://openlivewriter.org/tutorials/googlePhotoFix.html this should remedy your issue.

**Comment by drhamptn** [comment]
Hi, Scott, and thanks,

  I have created a new album, but still have the same problem: text only posts are published without problem, while ones with a single small picture return an error: The remote server returned an error: (500) Internal Server Error.

  I Removed and Added the blog link, re-booting OLW between, to be sure that things updated.  The album structure on Google Photos Albums is attached: the new album that I created per your instructions is the one with one item in it.   I made a post with a picture from the Google Blogger online editor and it posted fine, but I didn’t see that the photo went into any of the albums.

 I don’t want to break the links to prior photos, so have resisted renaming folders: I know that you are not a support person for my problems, but I’d be grateful for any other suggestions?

  Thanks, Dave
